Packs/JARM/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "JARM",
+    "description": "This pack introduces the JARM indicator type and commands for generating new JARM fingerprints of servers using TLS.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
```

Comment:
Please enter your url/email or leave this field empty

Reply:
resolved